google/cloud/bigquery_storage_v1beta2/proto/storage_pb2_grpc.py

```diff
@@ -123,3 +124,92 @@ def add_BigQueryReadServicer_to_server(servicer, server):
         "google.cloud.bigquery.storage.v1beta2.BigQueryRead", rpc_method_handlers
     )
     server.add_generic_rpc_handlers((generic_handler,))
+
+
+# This class is part of an EXPERIMENTAL API.
+class BigQueryRead(object):
```

Comment:
Do we know what these new objects are about? Seems like a grpc stub. Does it affect the client objects?

Reply:
I don't, as I'm not really that familiar with BQ storage. It's the same thing that can be seen in other autogen PRs, though, and at least one of them has been approved back then, thus I assume it's fine.

Reply:
Ok 👍 

Since the tests are passing, I assume we're good to go.